homeassistant/components/arlo/__init__.py

```diff
@@ -73,7 +76,7 @@ def setup(hass, config):
         )
         return False
 
-    def hub_refresh(event_time):
+    def hub_refresh(_: ServiceCall | datetime) -> None:
```

Comment:
The method is called from either the service registration, or via `track_time_interval` which seems to be using passing a `datetime` object as the first argument.